torch/csrc/jit/codegen/cuda/utils.h

```diff
@@ -297,7 +302,33 @@ SPECIALIZE_PRINTER(std::vector<int64_t>);
 
 #undef SPECIALIZE_PRINTER
 
-#endif
+#endif // if 0
+
+// Stringification with delimiter
+template <typename Iterator>
+std::string toDelimitedString(
+    Iterator first,
+    Iterator last,
+    std::string delim = ", ") {
+  std::stringstream ss;
+  bool first_val = true;
+  for (auto it = first; it != last; ++it) {
+    if (!first_val) {
+      ss << delim;
+    }
+    ss << Printer<typename std::remove_const<typename std::remove_reference<
+        decltype(*it)>::type>::type>::toString(*it);
```

Comment:
Should we just use `ss << *it`? `Printer` is not doing more than that.

Reply:
Hmm, I somehow thought `Printer` uses the `toString` method such as `Val::toString` when available. I'll remove the usage of `Print`.